### ISSUE TITLE
Show Group in Auto-Type

### ISSUE THREAD

**Issue by pavmik** [keeweb/keeweb#1441]
Is frustrating when I have more similar entries in different groups and is hard to find out to which group they belong.

Suggestion:
a) show group of entries in auto type window
b) new possibility to set which columns appear on the Auto Type popup windows (like in KeePass)

Thank You

**Comment by dploeger** [comment]
@antelle Can I help here? I'd like the same feature. :)

**Comment by antelle** [comment]
A simple solution would be to show the group. A bit more complicated variant of it is a possibility to select columns in the auto-type window, as it's suggested.

**Comment by dploeger** [comment]
As only some fields are available on every entry, I don't think, a sophisticated version of a list box with checkable fields would be feasible. A not so sophisticated version would be a free text field where the user can enter a comma separated list of fields to show but I don't like either. 

Perhaps adding the path to the entry (instead of only its group) and adding something like a quick view in autotype for the entry would be enough.

Adding the values of non-secured fields in the "more" popup would also be an idea.

**Comment by dploeger** [comment]
@pavmik what do you say?

**Comment by antelle** [comment]
Yes, something like what we have here:
<img width="591" alt="Untitled" src="https://user-images.githubusercontent.com/633557/85879613-f20e5300-b7da-11ea-8f32-0d97ba0c1bde.png">

**Comment by dploeger** [comment]
We can add a selection of those fields. However, I'd suggest a "path" instead of just the group, because I tend to have similar named subgroups.

**Comment by antelle** [comment]
It can be both, actually

**Comment by pavmik** [comment]
> Yes, something like what we have here:
> <img alt="Untitled" width="591" src="https://user-images.githubusercontent.com/633557/85879613-f20e5300-b7da-11ea-8f32-0d97ba0c1bde.png">

Yes, this would be nice on auto type window, and on "All Items" window with unchecked "Entries list table view".  With Group and/or full path...

![image](https://user-images.githubusercontent.com/6639625/86919440-86a47980-c128-11ea-9b24-dfb9f252db02.png)

**Comment by pavmik** [comment]
> Yes, something like what we have here:
> <img alt="Untitled" width="591" src="https://user-images.githubusercontent.com/633557/85879613-f20e5300-b7da-11ea-8f32-0d97ba0c1bde.png">

Yes, this would be nice on auto type window, and on "All Items" window with unchecked "Entries list table view".  With Group and/or full path...

![image](https://user-images.githubusercontent.com/6639625/86919673-e438c600-c128-11ea-8b65-8fea1457b181.png)
